source/chapters/25_sprites_and_walls/step_5.py

```diff
@@ -113,14 +113,14 @@ def on_update(self, delta_time):
         self.physics_engine.update()
 
         # Scroll the screen to the player
-        self.scroll_to_player()
+        #self.scroll_to_player()
```

Comment:
Yep, this is missing. The linked example from the API has the definition.

With the new 3.0, if it ever gets released, has a reworked camera that will change this section again.

Reply:
Is there something I can do to help you with version 3.0 ?